read/parser.go

```diff
@@ -171,9 +170,9 @@ type Edge struct {
 // object represents an object in the heap.
 // There will be a lot of these.  They need to be small.
 type object struct {
-	Ft     *FullType
-	offset int64 // position of object contents in dump file
-	Addr   uint64
+	Addr     uint64
+	Fields   []Field
```

Comment:
This will definitely lead to a larger `object` record.  That's the limiting factor for how big a heap dump this code can process.  Maybe there is a better way?